src/App.js

```diff
@@ -171,18 +173,76 @@ function App() {
     })
   }, [contactFormValues])
 
+
+  //Posting a new user to the Users api when Signing Up
+  const postNewUser = (newUser) => {
+    axios
+      .post('https://character-randomizer-backend.herokuapp.com/api/auth/register', newUser)
+      .then(res => {
+        console.log(`Response:`, res)
+        setUsers([...users, res.data.user])
```

Comment:
Do you need an array of users? Since you'll only have one user logged in at a time this state would be better as an object rather than an object of arrays